src/UI/MapView.java

```diff
@@ -40,9 +40,25 @@ protected void paintComponent(Graphics g) {
 			}
 		}
 
-		// Draw the current cursor location.
-		g.setColor(new Color(253,171,77));
-		g.fillRect(myGame.getCursorX()*tileSizePx, myGame.getCursorY()*tileSizePx, tileSizePx, tileSizePx);
+		if (myGame.currentMenu == null) {
+			// Draw the current cursor location.
+			g.setColor(new Color(253,171,77));
+			g.fillRect(myGame.getCursorX()*tileSizePx, myGame.getCursorY()*tileSizePx, tileSizePx, tileSizePx);
+		} else {
+			g.setColor(Color.black); // outer border
+			g.fillRect(myGame.gameMap.mapWidth*tileSizePx/4, myGame.gameMap.mapHeight*tileSizePx/4, myGame.gameMap.mapWidth*tileSizePx/2, myGame.gameMap.mapHeight*tileSizePx/2);
```

Comment:
Using the GameMap width and height to set the size of the menu will not work once we have different sized maps. Also, it would more readable to pull the value into a local variable instead of repeating a lengthy incantation for each usage.